sparse/CMakeLists.txt

```diff
@@ -139,6 +139,13 @@ KOKKOSKERNELS_GENERATE_ETI(Sparse_par_ilut_numeric par_ilut_numeric
   TYPE_LISTS  FLOATS ORDINALS OFFSETS LAYOUTS DEVICES
 )
 
+KOKKOSKERNELS_GENERATE_ETI(Sparse_gmres_numeric gmres_numeric
```

Comment:
Since gmres doesn't need a symbolic/numeric split, I would prefer to just call this "gmres".  Having "gmres_numeric" kind of implies there's a "gmres_symbolic" also